uint32 robot_id
GnssPosition position
uint32 error_gnss_cm
uint32 heading
float32 battery_voltage
MotorData front_left_wheel
MotorData front_right_wheel
MotorData back_right_wheel
MotorData back_left_wheel
MotorData left_mower
MotorData right_mower
float32 back_axle_angle
float32 sun_charge
bool wifi_is_up
float32 roll
float32 pitch
uint32 timestamp_seconds
uint32 timestamp_milliseconds
float32 gyroscope_x
float32 gyroscope_y
float32 gyroscope_z
float32 accelerometer_x
float32 accelerometer_y
float32 accelerometer_z
float32 magnetometer_x
float32 magnetometer_y
float32 magnetometer_z